tools/setup/emoji_dump/emoji_dump.py

```diff
@@ -8,18 +8,22 @@
 import sys
 import xml.etree.ElementTree as ET
 from six import unichr
+from os.path import abspath, dirname, join
```

Comment:
Stylistically, we generally try to avoid importing short names from `os.path` this way -- just use `os.path.dirname` directly.